alvin_c/server.js

```diff
@@ -0,0 +1,108 @@
+var express = require('express');
+var app = express();
+//
+var bodyParser = require('body-parser');
+app.use(bodyParser.urlencoded({ extended: true }));
+
+var path = require('path');
+app.use(express.static(path.join(__dirname, './static')));
+app.set('views', path.join(__dirname, './views'));
+
+app.set('view engine', 'ejs');
+
+// import and use mongoose
+var mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost/basic_meese');
+var MeeseSchema = new mongoose.Schema({
+ name: String,
+ age: Number
+})
+mongoose.model('Meese', MeeseSchema); // We are setting this Schema in our Models as 'meese'
+var Meese = mongoose.model('Meese') // We are retrieving this Schema from our Models, named 'meese'
+mongoose.Promise = global.Promise; // because reasons
+
+// Routes //
+// display all of the meese
+app.get('/', function(req, res) {
+  Meese.find({}, function(err, meese){
+    if(!err){
```

Comment:
We recommend to write 'truthy' statements. This is nothing more than a good coding habit that a lot of developers subscribe to. If(err) is strangely prefered to if(!err).